ex17_ch4_v2.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+#step1: write out constants
+BAC_ILLEGAL_TO_DRIVE = 0.08
+
+#step2: get input
+gender = input("Enter your gender, male or female: ")
+
+weight = int(input("Enter your weight in pounds: "))
+
+time = int(input("Enter the number of hours since your last drink. If it's been less than one hour enter 1: "))
+
+alcohol = int(input("Enter the amount of alcohol in ounces: "))
+
+#step3: set up two conditionals based on male or female response (since the ratio for the BAC equation depends on gender)
+if "male" in gender:
```

Comment:
You're comparing a string to "male", so you should use `if "male" == gender`. The `in` operator is best to check if an item is `in` a collection, such as a list or tuple.  Same applies to line 28.

Reply:
is it:
"male" == gender
or
gender == "male"?